src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/MemberProfileEditView.jsx

```
@@ -86,8 +86,8 @@ class MemberProfileEditView extends React.Component {
     if (errorMessage && errorMessage.length) {
       return (
         <Alert
-          autoDismiss={false}
           onDismiss={onClickDismiss}
+          timer={5000}
```

Why we are hiding the error message automatically? It should be dismissed by user.

---

QA reported a bug that it should be automatically hide after 5 secs, because the error remains untill user do not cancels it.